src/main/java/seedu/address/logic/commands/exceptions/CommandException.java

```diff
@@ -1,5 +1,7 @@
 package seedu.address.logic.commands.exceptions;
 
+import seedu.address.logic.commands.InternshipCommand;
```

Comment:
any reason for adding this since it is not being used?

Reply:
I need the import to write the JavaDocs for the class